lib/blocks.php

```diff
@@ -273,43 +273,22 @@ function gutenberg_register_core_block_assets( $block_name ) {
 	// and instead attach each block's theme-styles to their block styles stylesheet.
```

Comment:
Tiny nit: one line above this comment (Github wouldn't let me comment on it) the comment says "dequeue the full stylesheet". Looks like we can remove that part of the comment now?

Reply:
Oh well spotted! I'll update the comment.